src/components/main/Cards.js

```diff
@@ -0,0 +1,57 @@
+import "./Cards.css";
+import noCardImg from "../../assets/bg_noImg.png";
+
+function Cards({ cardData }) {
+  // 시간 차 구하기
+  const ElapsedTime = (elem) => {
+    const createdTime = new Date(elem.createdAt);
+    const currentTime = new Date();
+    const timeDiff = Math.floor((currentTime - createdTime) / 1000 / 60);
+
+    if (timeDiff < 2) return "1 minute ago";
+    else if (timeDiff <= 59) return `${timeDiff} minutes ago`;
+    else if (timeDiff <= 60 * 23)
+      return `${Math.floor(timeDiff / 60)} hours ago`;
+    else if (timeDiff <= 60 * 24 * 30)
+      return `${Math.floor(timeDiff / 24 / 60)} days ago`;
+    else if (timeDiff < 60 * 24 * 365)
+      return `${Math.floor(timeDiff / 30 / 24 / 60)} months ago`;
+    else return `${Math.floor(timeDiff / 30 / 24 / 60 / 12)} years ago`;
+  };
+
+  // 출력
+  return (
+    <div className="Cards">
+      <div className="container">
+        {cardData.map((elem) => (
```

Comment:
여기도 elem 이라는 변수명이 조금 모호하네요! 👀
딱히 변수명을 지을게 없다면 data 를 사용하는 편인 것 같아요.
혹은 위의 코멘트에서도 언급드렸다싶이, cardData 가 배열타입이니..아래와 같이 변경되면 더 명확해질 것 같아요!

```suggestion
        {cards.map((card) => (
```